config/webpack/webpack.config.js

```diff
@@ -218,7 +220,15 @@ const buildWebpackConfigs = builds.map(
       },
       {
         entry: {
-          render: paths.renderEntry || paths.serverEntry
+          render:
+            paths.renderEntry || path.join(__dirname, '../server/server.js')
+        },
+        resolve: {
+          alias: isRender
+            ? {}
+            : {
+                serverEntry: paths.serverEntry
```

Comment:
I feel like we need to use a more self-documenting alias that makes it obvious what's going on.

e.g. `__sku_alias__serverEntry`, so that when you see `require('__sku_alias__serverEntry')` it's a hint something different is going on.